ios/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -522,7 +522,11 @@ - (void)viewDidMount:(UIView *)view withBeforeSnapshot:(nonnull REASnapshot *)be
   }
 
   if (_hasAnimationForTag(viewTag, @"sharedElementTransition") && [type isEqual:@"entering"]) {
-    [_sharedTransitionManager notifyAboutNewView:view];
+    if ([type isEqual:@"entering"]) {
```

Comment:
It still doesn't make to me, in line 524 we already checked that `[type isEqual:@"entering"]`, why do we need to check the same condition in lint 525?